### ISSUE TITLE
Web Content Accessibility Guidelines (WCAG) Colour Contrast Checked

### ISSUE THREAD

**Issue by PavelGolodoniuc** [DevToys-app/DevToys#1407]
### What feature or new tool do you think should be added to DevToys?

W3C provided a set of [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/), which include a colour contrast checker currently integrated in  Chrome DevTools and Lighthouse, among others.

It would be great to have an extension or make it part of the core functionality in the DevToys.

Chrome DevTools does a good job when it works, including providing a colour curve of compliance with AA and AAA requirements. Unfortunately, it doesn't always work reliably as it fails to identify the background colour of an element or text block.

Some examples:

1. https://contrastchecker.com/
2. https://webaim.org/resources/contrastchecker/

### Why do you think this is needed?

A Colour Contrast Checker in the Graphic section of the DevToys.

**Comment by PavelGolodoniuc** [comment]
I just looked through the issues, and it appears that such a feature used to be present in DevToys v1.x (#875). Is it just a matter of porting that functionality to the v2?
